index.js

```diff
@@ -26,18 +26,27 @@ module.exports = {
     postcssOptions
   },
 
-  // TODO get this to work properly and remove the need for the default blueprint
+  //TODO: Not sure if we need this anymore since this lives in ember-cli-build now
```

Comment:
@mansona can you comment on whether we still need this?

Reply:
So we will need this and we should probably remove the code from the ember-cli-build file.

Ember-cli-build only works on the dummy app when working in an add-on, if we relied on only this they we would break a lot of downstream setups 🙈 does this make sense?

Reply:
Hmm, but ember-responsive-image changed where the config lives in v2, so wouldn't that impact us @mansona?

> The configuration of the addon moved from config/environment.js to ember-cli-build.js.

https://github.com/kaliber5/ember-responsive-image/blob/v2.0.0/docs/MIGRATION.md

Reply:
If this is the code I actually committed in ember-cli-build itself, I committed those so for future developers who work on it can run the dummy app and not get errors

Reply:
Basically we want to solve this issue before we merge this: https://github.com/empress/empress-blog-casper-template/pull/37

We want ember-responsive-image to be a dependency of the casper template without having to generate the config in the consuming app.

Reply:
Would you want to do that though? It seems like this project has to write the CSS to fit the images. If you pass this on to another app and they specify a config that is too big or small, it could break the templates. Or do you think this would not be an issue

Reply:
No, that's the point. Right now, today, apps can specify any config they want. The config lives in the app and is provided by the blueprint for the template. We want to bring that internally to this project so that the casper template controls the config and can have the correct settings for the layouts.

Reply:
Is that because it was in the config/env? Maybe now that is in cli build they cannot? 🤔

Reply:
This ember-cli-build file actually only runs for the demo app, it doesn't impact consumers. That's why, to maintain the existing functionality, you had to switch to that `applyBuildConfig` method in the blueprint. We basically want to get rid of that altogether and find a way for consumers to inherit a working config without writing anything to the consuming projects where this is installed.